share/dictionary/radius/dictionary.huawei

```diff
@@ -189,12 +189,12 @@ VALUE	Port-Mirror			Enable			3
 
 ATTRIBUTE	Account-Info				184	string
 ATTRIBUTE	Service-Info				185	string
-ATTRIBUTE	DHCP-Option				187	octets
+ATTRIBUTE	DHCP-Option				187	string
```

Comment:
This should be left at `octets`, or even `group ref=..DHCPv4` as per the RFC 9445 dictionary, and the `DHCP-Options` attribute.